lib/utils/cached_secure_storage.dart

```diff
@@ -23,10 +23,14 @@ class CachedSecureStorage {
     if (value != null) {
       await _flutterSecureStorage.write(key: key, value: value);
```

Comment:
We should remove the null check and call delete method directly if we want to remove a key instead of passing null.